containers/server-saline-image/cleanup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+sed -i '/pam_systemd.so/d' /etc/pam.d/*
+
+rpm -e --nodeps \
+  systemd systemd-presets-branding-openSUSE systemd-default-settings-branding-openSUSE systemd-default-settings systemd-presets-common-SUSE \
+  curl container-suseconnect kubic-locale-archive file-magic libmagic1 dmidecode netcfg update-alternatives \
+  zypper libzypp boost-license1_66_0 libboost_thread1_66_0 libsolv-tools libsolv-tools-base \
+  info xz gzip perl-base ncurses-utils libsqlite3-0 \
+  libsodium23 iproute2 p11-kit-tools libp11-kit0 cracklib-dict-small \
+  libxtables12 libcap-ng0 libcap2 libapparmor1 procps libprocps8 libusb-1_0-0 \
+  python3-Babel python3-zypp-plugin python3-rpm python3-pyasn1 python3-asn1crypto python3-urllib3 python3-idna \
+  python3-pyparsing python3-pycparser python3-setuptools python3-cryptography python3-M2Crypto python3-psutil \
+  python3-cffi python3-chardet python3-pyOpenSSL python3-py python3-requests python3-ply python3-pytz python3-jmespath \
+  python3-appdirs python3-apipkg  \
+  rpm-config-SUSE rpm-ndb
+
+rm -rf /var/lib/zypp /var/log/zypp /var/cache/zypp /var/log/* /var/lib/rpm/*
```

Comment:
Nope, because there are a list of the hard dependencies for `python3-salt` which makes sense for either master or minion, but doesn't make any sense for Saline as it's not using such calls internally. As the result on installing python3-salt it installs all the dependencies, but maybe threre is any way to install the packages with forcing to ignore the dependencies completely.

Reply:
This thread might be helpful: https://forums.opensuse.org/t/how-to-install-software-without-dependencies-in-zypper/20716/7

Reply:
I changed the approach to make it possible to have more clear installation to prevent redundant dependencies in a kind of mixed way from the latest suggestion, but still using zypper to get the require packages. It allows to get way smaller image. Hope this approach is more straight, but multistage in this case makes it overcomplicated, for me looks like this way is easier anyway.